jxl/src/image.rs

```diff
@@ -498,8 +498,7 @@ impl<'a, T: ImageDataType> ImageRectMut<'a, T> {
     }
 }
 
-#[cfg(feature = "debug_tools")]
-pub mod debug_tools {
+pub mod output_tools {
```

Comment:
Move this to `enc`?

Do we even need the `debug_tools` feature anymore?

(in the long term, this functionality should probably go in a separate crate...)

Reply:
Moved these to enc, also removed the debug_tools feature